src/measure_theory/simple_func_dense.lean

```diff
@@ -458,6 +457,16 @@ lemma measure_preimage_lt_top_of_integrable (f : α →ₛ E) (hf : integrable f
   μ (f ⁻¹' {x}) < ∞ :=
 integrable_iff.mp hf x hx
 
+lemma measure_lt_top_of_mem_ℒp_indicator (hp_pos : 0 < p) (hp_ne_top : p ≠ ∞) {c : E} (hc : c ≠ 0)
+  {s : set α} (hs : measurable_set s)
+  (hcs : mem_ℒp ((const α c).piecewise s hs (const α 0)) p μ) :
+  μ s < ⊤ :=
+begin
+  have : function.support (const α c) = set.univ := function.support_const hc,
+  simpa only [mem_ℒp_iff_fin_meas_supp hp_pos hp_ne_top, fin_meas_supp_iff_support,
+    support_indicator, set.inter_univ, this] using hcs
+end
```

Comment:
Does the following work?
```suggestion
by simpa only [mem_ℒp_iff_fin_meas_supp hp_pos hp_ne_top, fin_meas_supp_iff_support,
    support_indicator, set.inter_univ, function.support_const hc] using hcs
```

Reply:
No ... the type annotation is needed for Lean to discover `α`.